salt/states/zk_concurrency.py

```diff
@@ -126,3 +131,30 @@ def unlock(path,
         ret['comment'] = 'Unable to find lease for path {0}'.format(path)
 
     return ret
+
+
+def min_party(path,
```

Comment:
Ah, yes, the first state argument should always be name, you made it `path`... that's why it triggered your warnings...